packages/flutter/lib/src/widgets/platform_view.dart

```diff
@@ -580,3 +580,50 @@ class _UiKitPlatformView extends LeafRenderObjectWidget {
     renderObject.updateGestureRecognizers(gestureRecognizers);
   }
 }
+
+/// A helper widget provides the implementer of a new platform view certain utilities.
+///
+/// The widget is responsible to handle painting, gestures and semantics.
```

Comment:
Remove - that'd be more true of the render object anyway.

It would be more helpful here to have some prose explaining what PlatformView widgets have to do, or why you'd wnat to compose with this widget.